bodhi/tests/server/scripts/test_check_policies.py

```diff
@@ -59,6 +59,35 @@ def test_policies_satisfied(self):
                                                expected_query)
 
     @patch.dict(config, [('greenwave_api_url', 'http://domain.local')])
+    def test_policies_pending_satisfied(self):
+        """Assert correct behavior when the policies enforced by Greenwave are satisfied"""
```

Comment:
This comment doesn't quite capture the purpose of the test as I understand it - can we reword it to state that we are ensuring that pending updates use the ```bodhi_update_push_testing``` decision context? That way it'll be clearer what this test is for later.